client/src/App.js

```diff
@@ -15,19 +16,22 @@ import "./App.css";
 class App extends Component {
   render() {
     return (
-      <Router>
+      <Switch>
```

Comment:
@simonnguyen3054 Just wondering why you changed the tag from `<Router>` to `<Switch>`?

Reply:
so Switch just to group the relevant routes together. It's not really make a difference in this case but I did it anyway so later we can use it.